atlp-rwanda/strikers-bn-be
# Ft users should be able to book a room in accomodationfacility 182215595

#### What does this PR do?

- Complets all endpoints for Companies and Bookings, and their tests

#### Description of Task to be completed?

- Completes implementing all APIs for Bookings and Companies management, and also includes unit tests for all those endpoints

#### How should this be manually tested?

- Clone the repo

- checkout to branch `ft-users-should-be-able-to-book-a-room-in-accomodationfacility-182215595`

- Run `npm install` to install dependencies

- Run `npm run test` to run tests

- Run `npm run coverage`

#### Pivotal Tracker Story

- [Link to Pivotal Tracker Story](https://www.pivotaltracker.com/n/projects/2569362/stories/182215595)
